### PR TITLE
Update `k3d` completions

### PR DESCRIPTION
**Pull request by dkravetz**
## Description

Update `k3d completions to match the stable 3.0.0 release and include completion for subcommands.

For example `k3d cluster start` will suggest the name of existing k3d clusters

### CODE REVIEW

File: share/completions/k3d.fish [hunk 1]
```diff
@@ -1,13 +1,58 @@
...
+            return 1
+        end
+    end
+    return 0
+end
+
+function __fish_k3d_print_clusters --description 'Print a list of k3d clusters'
+    for key in (k3d cluster list --no-headers | awk '{print $1}')
+        echo $key
+    end
```
[the remark this answers is not in the document]
Comment (dkravetz) [reply share/completions/k3d.fish:13]: I has having some issues before and wrongly attributed them to this, thanks!

File: share/completions/k3d.fish [hunk 2]
```diff
@@ -1,13 +1,58 @@
...
+complete -f -c k3d -n '__fish_k3d_no_subcommand' -a node -d "Manage node(s)"
+
+complete -f -c k3d -n '__fish_seen_subcommand_from cluster' -a create -d "Create a new cluster"
+complete -f -c k3d -n '__fish_seen_subcommand_from cluster' -a list -d "List cluster(s)"
+complete -f -c k3d -n '__fish_seen_subcommand_from cluster' -a delete -d "Delete cluster(s)"
+complete -f -c k3d -n '__fish_seen_subcommand_from cluster' -a start -d "Start existing k3d cluster(s)"
+complete -f -c k3d -n '__fish_seen_subcommand_from cluster' -a stop -d "Stop existing k3d cluster(s)"
+complete -f -c k3d -n '__fish_seen_subcommand_from cluster; and __fish_seen_subcommand_from delete' -a '(__fish_k3d_print_clusters)' -d "Cluster"
+complete -f -c k3d -n '__fish_seen_subcommand_from cluster; and __fish_seen_subcommand_from start' -a '(__fish_k3d_print_clusters)' -d "Cluster"
+complete -f -c k3d -n '__fish_seen_subcommand_from cluster; and __fish_seen_subcommand_from stop' -a '(__fish_k3d_print_clusters)' -d "Cluster"
```
[the remark this answers is not in the document]
Comment (dkravetz) [reply share/completions/k3d.fish:37]: If I merge this commands into both the `cluster` and the `node` sections, whenever I type either `k3d cluster` or `k3d node` followed by any of `start`, `stop`, or `delete`, the completion suggests both nodes and clusters. This doesn't happen with `k3d kubeconfig get`, however.

[This is a imgur link](https://imgur.com/a/nCyiF9I) to a screenshot showing this behaviour.

I'm not going to lie, I got tired of this behavior and went for a more simplistic, but repetitive approach :)

[the remark this answers is not in the document]
Comment (dkravetz) [reply share/completions/k3d.fish]: Nevermind, it looks like I couldn't process propositional logic after a long day :joy:

**Comment by krobelus** [comment]
Thank you, merged.

**Pull request merged**